pilot/pkg/security/authz/builder/testdata/v1alpha1-simple-policy-td-aliases-out.yaml

```diff
@@ -0,0 +1,30 @@
+rules:
+  policies:
+    httpbin-viewer:
+      permissions:
+        - andRules:
+            rules:
+              - orRules:
+                  rules:
+                    - header:
+                        exactMatch: GET
+                        name: :method
+      principals:
```

Comment:
Could you address https://github.com/istio/istio/pull/17594/files#r334550461 and generate the out file again?

Reply:
Done. PTAL. Thanks!